project/Dependencies.scala

```diff
@@ -190,11 +184,7 @@ object Dependencies {
   // google2 lib requires specific versions of rpc libs:
```

Comment:
Presumably this comment is wrong then?  google2 lib no longer requires specific versions?